backend/pkg/api/handlers/public.go

```diff
@@ -1323,26 +1318,40 @@ func (h *HandlerService) PublicGetValidatorDashboardGroupRewards(w http.Response
 //	@Tags			Validator Dashboard
 //	@Produce		json
 //	@Param			dashboard_id	path		string	true	"The ID of the dashboard."
+//	@Param			group_ids		query		string	false	"Provide a comma separated list of group IDs to filter the results by."
+//	@Param			aggregation		query		string	false	"Aggregation type to get data for."	Enums(epoch, hourly, daily, weekly)	Default(hourly)
 //	@Param			modes			query		string	false	"Provide a comma separated list of protocol modes which should be respected for validator calculations. Possible values are `rocket_pool``."
+//	@Param			after_ts		query		string	false	"Return data after this timestamp."
+//	@Param			before_ts		query		string	false	"Return data before this timestamp."
```

Comment:
Thought: Maybe we should investigate whether this interval is really open or closed. Though this depends on how the handler layer determines the limits, how data access queries the data and whether the data stored in our db has inclusive/exclusive start and end epochs. Might not be worth the effort though.

Reply:
Good point, this should probably be documented more explicitly and handled carefully. Currently it's inclusive on both ends, but it's the same for the summary charts endpoint (https://github.com/gobitfly/beaconchain/blob/BEDS-875/rewards-chart/backend/pkg/api/data_access/vdb_summary.go#L1049). I don't really mind which way it should be, do you have a preference? Otherwise could ask frontend/consumers